spark/src/test/java/org/opensearch/sql/spark/dispatcher/SparkQueryDispatcherTest.java

```diff
@@ -936,6 +936,11 @@ void testDispatchQueryWithExtraSparkSubmitParameters() {
 
   private String constructExpectedSparkSubmitParameterString(
       String auth, Map<String, String> authParams, String query) {
+    return constructExpectedSparkSubmitParameterString(auth, authParams, query, false);
+  }
+
+  private String constructExpectedSparkSubmitParameterString(
```

Comment:
Can we also add a unit test with  lake formation enabled datasource.

Reply:
oops. meant to add one and forgot. Added in latest updatel